ksp_physics/ksp_physics_lib.hpp

```diff
@@ -26,6 +26,12 @@ PHYSICS_DLL_TEMPLATE_CLASS internal_continuous_trajectory::ContinuousTrajectory<
     ksp_plugin::Barycentric>;
 PHYSICS_DLL_TEMPLATE_CLASS
     internal_ephemeris::Ephemeris<ksp_plugin::Barycentric>;
+PHYSICS_DLL_TEMPLATE_CLASS
+    internal_ephemeris::Ephemeris<
+        ksp_plugin::Barycentric>::AdaptiveStepParameters;
+PHYSICS_DLL_TEMPLATE_CLASS
+    internal_ephemeris::Ephemeris<
+        ksp_plugin::Barycentric>::GeneralizedAdaptiveStepParameters;
```

Comment:
`AdaptiveStepParameters` isn't a template class yet, I don't think this will work.